### PR TITLE
[4.0] Solving Duplicate ID ( #25088 ) Multilingual

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue #25088 .

### Summary of Changes
In https://github.com/joomla/joomla-cms/pull/22166 we solved com_associations side by side "Bug1" by adding `id` to `$hiddenFields` in `layouts/joomla/edit/global.php`.
This only concerned articles when `Show Publishing Options` was set to `Hide` in Articles Options.

As found out in #25088, this creates a duplicate ID.
Example: see source in `administrator/index.php?option=com_tags&view=tag&layout=edit`
To solve the issue, this PR adds the `$hidden_fields` directly in the article edit tmpl and takes it off from global.php.

### Testing Instructions
Patch a clean install. (This can be done using Patchtester on a nighlty build)
Create a multilingual site by installing 2 languages and using the Multilingual Sample Data module in CPanel.
Set `Show Publishing Options` to `Hide` in Articles Options.
Load com_associations and set the filters to Articles and one of the content languages.
Click on the title of one of the articles to make sure the side by side will not loop.
Load `administrator/index.php?option=com_tags&view=tag&layout=edit` and verify that we have only one `jform_id`

### After patch
All still works fine for side by side in com_associations.
No more duplicate ID

### CODE REVIEW

**Comment by brianteeman** [comment]
After applying the PR and going to the article edit form 

image

**Comment by infograf768** [comment]
You are right. Looking into this now.

**Comment by infograf768** [comment]
@brianteeman 
Can you test again? I have put the echo in a hidden `<div>`
Did not find another way.

**Comment by brianteeman** [comment]
Sorry but this is not correct as there is still the ID in the publishing tab so if that is enabled you still get the duplicate id

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 0ec4a77d9c48bbcadd8f589f62a04bd49610929e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25096">issues.joomla.org/tracker/joomla-cms/25096</a>.</sub>

**Comment by infograf768** [comment]
> this is not correct as there is still the ID in the publishing tab so if that is enabled you still get the duplicate id

Correct. Added now conditional to echo the hiddenfield depending on Publishing Options.

Please test again.

File: administrator/components/com_content/tmpl/article/edit.php [hunk 1]
```diff
@@ -124,6 +124,12 @@
 
 		<?php echo HTMLHelper::_('uitab.endTabSet'); ?>
 
+		<?php // Creating 'id' hiddenField to cope with com_associations sidebyside loop ?>
+		<?php if ($params->get('show_publishing_options', 1) == 0) : ?>
+			<?php $hidden_fields = $this->form->getInput('id'); ?>
```
Comment (Quy) [inline administrator/components/com_content/tmpl/article/edit.php:131]: No need to assign to a variable. Do it like lines 134-135.

Comment (infograf768) [reply administrator/components/com_content/tmpl/article/edit.php:131]: @Quy
Please propose code.
I tried
```
		<?php if ($params->get('show_publishing_options', 1) == 0) : ?>
			<input type="hidden" name="hidden_fields" value="<?php echo $this->form->getInput('id'); ?>">
		<?php endif; ?>
```
but it does not work for the side by side in com_associations

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 5c85a91eeec8709a7a7f54cb9cd6ea5748680e5f<br /><br />I wont pretend to understand it but it works<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25096">issues.joomla.org/tracker/joomla-cms/25096</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b93ba3f58dc937c676072ae03d977ac9465a0976<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25096">issues.joomla.org/tracker/joomla-cms/25096</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25096">issues.joomla.org/tracker/joomla-cms/25096</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**